pkg/action/executor/pod_quota_evaluator.go

```diff
@@ -21,16 +21,13 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation"
 	quota "k8s.io/apiserver/pkg/quota/v1"
 	"k8s.io/apiserver/pkg/quota/v1/generic"
-	"k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/component-base/featuregate"
 )
 
 // the name used for object count quota
 var podObjectCountName = generic.ObjectCountQuotaResourceNameFor(corev1.SchemeGroupVersion.WithResource("pods").GroupResource())
 
 const (
-	// Allow specifying NamespaceSelector in PodAffinityTerm.
-	PodAffinityNamespaceSelector featuregate.Feature = "PodAffinityNamespaceSelector"
 	// Enables PodOverhead, for accounting pod overheads which are specific to a given RuntimeClass
 	PodOverhead featuregate.Feature = "PodOverhead"
```

Comment:
Please remove this also. Its not used anywhere.

Reply:
Done in this commit https://github.com/turbonomic/kubeturbo/pull/658/commits/a6bbaaa470f0535abe7179deb355893ccd711c82  @irfanurrehman

Reply:
Thanks @kevinwangcn